### PR TITLE
Sdk automl video object tracking batch

### PR DESCRIPTION
**Pull request by sudarshan-SpringML**
This PR contains notebook which demonstrates how to use the Vertex SDK to create video object tracking models and do batch prediction using a Google Cloud AutoML model
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/automl/sdk_automl_video_object_tracking_batch.ipynb [hunk 1]
```diff
@@ -0,0 +1,1023 @@
+{
```
Comment (kweinmeister) [inline notebooks/official/automl/sdk_automl_video_object_tracking_batch.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #18.        # Delete the AutoML or Pipeline trainig job
</pre><p>Typo: should be training</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

[the remark this answers is not in the document]
Comment (sudarshan-SpringML) [reply notebooks/official/automl/sdk_automl_video_object_tracking_batch.ipynb:1]: <p>resolved</p>

[the remark this answers is not in the document]
Comment (sudarshan-SpringML) [reply notebooks/official/automl/sdk_automl_video_object_tracking_batch.ipynb:1]: <p>resolved</p>

[the remark this answers is not in the document]
Comment (sudarshan-SpringML) [reply notebooks/official/automl/sdk_automl_video_object_tracking_batch.ipynb:1]: <p>resolved</p>

[the remark this answers is not in the document]
Comment (sudarshan-SpringML) [reply notebooks/official/automl/sdk_automl_video_object_tracking_batch.ipynb:1]: <p>resolved</p>

[the remark this answers is not in the document]
Comment (sudarshan-SpringML) [reply notebooks/official/automl/sdk_automl_video_object_tracking_batch.ipynb:1]: <p>resolved</p>

[the remark this answers is not in the document]
Comment (sudarshan-SpringML) [reply notebooks/official/automl/sdk_automl_video_object_tracking_batch.ipynb:1]: <p>resolved</p>

**Comment by sudarshan-SpringML** [comment]
Comments resolved. Notebook can be reviewed

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**